android/app/src/main/java/org/fossasia/openevent/fragments/ScheduleFragment.java

```diff
@@ -41,7 +41,6 @@
 
     @BindView(R.id.viewpager) ViewPager viewPager;
     @BindView(R.id.tabLayout) TabLayout scheduleTabLayout;
-    @BindView(R.id.schedule_fab_filter) FloatingActionButton fabFilter;
```

Comment:
Why are you removing its global declaration and initializing variable multiple times in separate functions?